chat-plugins/economy.js

```diff
@@ -164,14 +164,14 @@ exports.commands = {
 				Economy.writeMoney(targetUser, amount, () => {
 					Economy.readMoney(targetUser, firstAmount => {
 						Economy.readMoney(user.userid, secondAmount => {
-							this.popupReply("You sent " + amount + ((amount === 1) ? " currency " : " currency ") + " to " + targetUser);
```

Comment:
Why wern't we using `global.currencyName` and `global.currencyPlural` here? Weird, we should fix that.